tsfresh/__init__.py

```diff
@@ -18,6 +18,18 @@
     __version__ = 'unknown'
 
 
+# Set default logging handler to avoid "No handler found" warnings.
+import logging
+try:  # Python 2.7+
+    from logging import NullHandler
```

Comment:
ok, I understand. How would you store the information, that the logging message was already shown?